doc-site.html

```diff
@@ -180,6 +185,50 @@ <h2>Alert</h2>
 
                 </div>
 
+
+                     <!-- Nav bar component -->
+
+            <div class="component" id="navbar">
+                <h2>Navigation Bars</h2>
+                <p class="description">Navigation bar top essential element on any page to follow the users from
+                    sites
+                    one page to another page flawlessly. These nav are responsive and will have hamburger menu below
+                    768px</p>
+
+                <!-- Dark Nav bar-->
+                <h3>Dark Nav Bar</h3>
+                <nav class="nav nav-dark" style="position: relative;">
+                    <h3>BRAND</h3>
+                    <button class="hamburger"><i class="fa fa-bars fa-2x" aria-hidden="true"></i></button>
+                    <ul class="nav-list">
+                        <li>Home</li>
+                        <li>About</li>
+                        <li>Login</li>
+                    </ul>
+                </nav>
+
+                <h3>Light Nav Bar</h3>
+                <nav class="nav nav-light" style="position: relative;">
+                    <h3>BRAND</h3>
+                    <button class="hamburger"><i class="fa fa-bars fa-2x" aria-hidden="true"></i></button>
+                    <ul class="nav-list">
+                        <li>Home</li>
+                        <li>About</li>
+                        <li>Login</li>
+                    </ul>
+                </nav>
+                <button class="bttn bttn-secondary" onclick="showCodeFunctionality('navbar-code')">Code
+                    &lt/&gt
```

Comment:
Everything clean. You haven't written any JS code yet. On rendering, it is looking great.